sympy/assumptions/sathandlers.py

```diff
@@ -198,6 +199,17 @@ class ClassFactRegistry(FactRegistry):
     >>> reg(Abs(x))
     {Q.nonnegative(Abs(x)), Equivalent(~Q.zero(x), ~Q.zero(Abs(x)))}
 
+    Multiple facts can be registered at once by letting handler return an
+    iterable.
```

Comment:
I think it's better to have a separate function for registering multiple handlers rather than checking based on `iterable`.

Reply:
How would it be possible, when we are registering a single function `f`?

Reply:
Oh, maybe we can have a seperate container which stores these kind of functions.

Reply:
There are 4 situations:
1. Register a fact to a class
2. Register a fact to classes
3. Register facts to a class
4. Register facts to classes

Instead of having 4 methods, would it make more sense to have one method with argument(s) for classes and option for single/multiple facts?

Reply:
Maybe we just need one that can register a single fact to a single class and another that registers multiple facts to multiple classes.

Reply:
Done.